content/getting_started/Faq.md

```diff
@@ -35,6 +35,18 @@ opt input.ll -load-pass-plugin=/path/to/LLVMEnzyme-VERSION.so -passes=enzyme -o
 
 If you are using CMake, Enzyme exports a special `ClangEnzymeFlags` target which will automatically add the correct flags. See [here](https://github.com/EnzymeAD/Enzyme/blob/main/enzyme/test/test_find_package/CMakeLists.txt#L14) for an example.
 
+## I receive an `__enzime_autodiff` undefined symbol either at runtime or at compile time
+This is typically due to forgetting to include either or both `-fuse-ld=lld` and `-flto` in all the files involved in the creation of the binary.
```

Comment:
Maybe start by specifying that `This means that you have enzyme calls which didn't run the the Enzyme AD transformation. Please check that you have Enzyme run as part of compilation which resulted in the error. If using LLDEnzyme and LTO, make sure to include both `-fuse-ld=lld` and `-flto` in all the files involved ....`